blogposts/universal-code-intelligence-and-navigation-github.md

```diff
@@ -0,0 +1,43 @@
+---
+title: Universal Code Intelligence for GitHub with the Sourcegraph browser extension
+description: Universal Code Intelligence for GitHub means bringing code navigation features such as hover tooltips, go to definition, and find references to every code view and pull request, supporting every popular language, and all public and private repositories on both GitHub.com and GitHub Enterprise.
+author: Ryan Blunden
+authorUrl: https://twitter.com/ryan_blunden
+publishDate: 2020-04-07T00:00-15:30
+tags: [blog]
```

Comment:
We should start adding tags for future categorization.
```suggestion
tags: [blog, Github,"code intelligence",,"code navigation","private repositories"]
```

Reply:
I think we need a list of supported tags first but will make a note to come back and update this once we do.